static/src/javascripts/lib/formInlineLabels.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+import bean from 'bean';
+import $ from 'lib/$';
+import fastdom from 'fastdom';
+/**
+ * @param {String} type 'add' or 'remove'
+ * @param {Object} $el $ element
+ * @param {String} cssClass The class to add or remove
+ * @param {Function} testFunc A callback function that should return boolean describing whether we
+ * should perform the add or remove class
+ */
+const updateClass = (type, $el, cssClass, testFunc) =>
```

Comment:
these ought to be flowtyped if possible – we're trying to make it a rule than any 'platform' modules, like `lib`, are typed